modules/services/lorri.nix

```diff
@@ -0,0 +1,66 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.lorri;
+in
+
+{
+  meta.maintainers = [ maintainers.gerschtli ];
+
+  options = {
+    services.lorri = {
+      enable = mkEnableOption "lorri setup";
+
+      package = mkOption {
```

Comment:
Not particularly important but do you forsee this option being frequently used? If not then I would suggest removing and letting people override the package using overlays instead.

Reply:
You are right, removed this option